example/screens/PreviewScreen.tsx

```diff
@@ -106,6 +121,10 @@ const PreviewScreen = ({navigation}: Props) => {
           onPress={switchCamera}
           accessibilityLabel={SWITCH_CAMERA_BUTTON}
         />
+        <InCallButton
+          iconName="volume-high"
```

Comment:
this icon may suggest turning sound on/off, not selecting a device? maybe use "headphones" icon?

Reply:
Probably the best solution would be create button without any icon and then for every option adjust icon, but this is not in scope of example app right now. For now i would leave it as it is.